README.md

```diff
@@ -2,34 +2,38 @@
 
 [![Build Status](https://travis-ci.org/onnx/onnx-coreml.svg?branch=master)](https://travis-ci.org/onnx/onnx-coreml)
 
-This tool converts [ONNX](https://onnx.ai/) models to Apple Core ML format. To convert Core ML models to ONNX, use [ONNXMLTools](https://github.com/onnx/onnxmltools).
+## PyTorch Models
```

Comment:
We should put a disclaimer right on top that this package is no longer supported.

Reply:
Done.